.changeset/every-brooms-play.md

```diff
@@ -0,0 +1,5 @@
+---
+"kilo-code": patch
+---
+
+Minor improvements to autocomplete internal state handling
```

Comment:
in a sense this is the major releasenote, but since i cannot reproduce i cannot be sure we fix anything (but wanted to have a releasenote for traceability)